client/components/Facilitator/Components/Cohorts/CohortDataTable.jsx

```diff
@@ -424,73 +386,194 @@ export class CohortDataTable extends React.Component {
+                        <Icon name="microphone" />
                     </React.Fragment>
                 ) : (
-                    response
+                    content
                 );
 
-                const className = !response
+                const className = !content
                     ? 'cohortdatatable__cell-data-missing'
                     : '';
-                const name = 'detail';
-                const onCellClick = event =>
-                    detailOpen(event, {
-                        name,
-                        subject,
-                        prompt: headers[index].prompt,
-                        response: display
-                    });
 
                 return (
-                    <Table.Cell
-                        className={className}
-                        key={cellKey}
-                        name="cell"
-                        onClick={onCellClick}
-                        style={{
-                            cursor: 'pointer'
-                        }}
-                    >
-                        {display}
-                    </Table.Cell>
+                    <CohortDataModal
+                        {...props}
+                        index={cellIndex}
+                        key={modalKey}
+                        trigger={
+                            <Table.Cell
+                                className={className}
+                                key={cellKey}
+                                name="cell"
+                                style={{
+                                    cursor: 'pointer'
+                                }}
+                                verticalAlign="top"
+                            >
+                                <p>{display}</p>
+
+                                {display && (
+                                    <Moment
+                                        duration={response.created_at}
+                                        date={response.ended_at}
+                                        style={{
+                                            color: 'grey'
+                                        }}
+                                    />
+                                )}
+                            </Table.Cell>
+                        }
+                    ></CohortDataModal>
                 );
             })}
         </Table.Row>
     );
 };
 
+const CohortDataModal = props => {
+    const { index, prompts, rows } = props;
+    const component = prompts[index];
+    const { header, prompt, slide } = component;
+
+    return (
+        <Modal
+            trigger={props.trigger}
+            size="fullscreen"
+            className="cohortdatatablemodal__view"
+            closeIcon
+        >
+            <Modal.Header>Responses In Context</Modal.Header>
+
+            <Modal.Content className="cohortdatatablemodal__scroll">
+                <Grid columns={2} class="cohortdatatablemodal__grid-nowrap">
```

Comment:
whoops, `class` should be `className` here as it's a React property rather than the DOM one.

Reply:
Fixed